contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -469,6 +554,36 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         return _calculateCurrentLeverageRatio(currentLeverageInfo.collateralValue, currentLeverageInfo.borrowValue);
     }
 
+    /**
+     * Calculates the total notional rebalance size.
+     *
+     * @return uint256      total notional rebalance size
+     */
+    function getTotalRebalanceNotional()  external view returns(uint256) {
+
+        uint256 newLeverageRatio;
+        uint256 currentLeverageRatio = getCurrentLeverageRatio();
+
+        if (currentLeverageRatio > incentive.incentivizedLeverageRatio) {
+            newLeverageRatio = methodology.maxLeverageRatio;
+        } else if (twapLeverageRatio > 0) {
+            if (_isAdvantageousTWAP(currentLeverageRatio)) {
+                return 0;
+            }
+            newLeverageRatio = twapLeverageRatio;
+        } else {
+            newLeverageRatio = _calculateNewLeverageRatio(currentLeverageRatio);
+        }
+
+        bool isLever = newLeverageRatio > currentLeverageRatio;
```

Comment:
We don't need to create this variable if we directly do `newLeverageRatio > currentLeverageRatio` in line 581